warplib/src/conversions/integer_conversions.cairo

```diff
@@ -51,26 +51,6 @@ fn bool_into_felt252(val: bool) -> felt252 {
     }
 }
 
-// Cairo-1.1 corelib code from the RC
-// FIXME: remove, when updated
-impl U256TryIntoFelt252 of TryInto<u256, felt252> {
-    fn try_into(self: u256) -> Option<felt252> {
-        let FELT252_PRIME_HIGH = 0x8000000000000110000000000000000_u128;
-        if self.high > FELT252_PRIME_HIGH {
-            return Option::None(());
-        }
-        if self.high == FELT252_PRIME_HIGH {
-            // since FELT252_PRIME_LOW is 1.
-            if self.low != 0 {
-                return Option::None(());
-            }
-        }
-        Option::Some(
-            self.high.into() * 0x100000000000000000000000000000000_felt252 + self.low.into()
-        )
-    }
-}
```

Comment:
Does warplib tests work? It seems they're not run by the CI. I believe an import of try into implementation should be added

Reply:
Yes the run during the compilation stage. The implementation of the `Try` is now on the corelib by default, so need to have it here as well